src/Service/ExcelWriter.php

```diff
@@ -175,7 +175,13 @@ public function prepare($cacheFile, $sortedHeaders, $freezeHeaders = false)
         $file = new \SplFileObject($cacheFile);
         while (!$file->eof()) {
             $dataRow = json_decode($file->current(), true);
-            $this->writeRow($dataRow, $sortedHeaders);
+
+            if (is_string($dataRow) && $dataRow == '---') {
```

Comment:
Document what `---` is and just extract out it into a const

Reply:
Also document what you are doing in this block. We can just create a generic method for this like "Add Spacer" or "Add Section End" or something